src/cmd/src/metasrv.rs

```diff
@@ -137,6 +138,9 @@ struct StartCommand {
     /// The max operations per txn
     #[clap(long)]
     max_txn_ops: Option<usize>,
+    /// The database backend.
+    #[clap(long, value_enum)]
+    backend: Option<BackendImpl>,
```

Comment:
```suggestion
    #[clap(long, value_enum, default_value = "BackendImpl::EtcdBackend")]
    backend: BackendImpl,
```

Reply:
actually this change will make the fuzzy test fail with metaserver crashing due to this is not an optional field. Does clip supply the default enum value when the field is not present?